modules/user-groups/user-groups.php

```diff
@@ -579,83 +619,98 @@ function print_configure_view() {
 			</script>
 			<div id="col-right"><div class="col-wrap">
 				<?php $wp_list_table->display(); ?>
 			</div></div>
 			<div id="col-left"><div class="col-wrap"><div class="form-wrap">
 				<h3 class="nav-tab-wrapper">
-					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab<?php if ( !isset( $_GET['action'] ) || $_GET['action'] != 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Add New', 'edit-flow' ); ?></a>
-					<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab<?php if ( isset( $_GET['action'] ) && $_GET['action'] == 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Options', 'edit-flow' ); ?></a>
+					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab
+										<?php
+										if ( ! isset( $_GET['action'] ) || 'change-options' != $_GET['action'] ) {
+											echo ' nav-tab-active';}
+										?>
+					"><?php _e( 'Add New', 'edit-flow' ); ?></a>
+					<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab
+										<?php
+										if ( isset( $_GET['action'] ) && 'change-options' == $_GET['action'] ) {
+											echo ' nav-tab-active';}
+										?>
+					"><?php _e( 'Options', 'edit-flow' ); ?></a>
 				</h3>
-				<?php if ( isset( $_GET['action'] ) && $_GET['action'] == 'change-options' ): ?>
+				<?php if ( isset( $_GET['action'] ) && 'change-options' == $_GET['action'] ) : ?>
 				<form class="basic-settings" action="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" method="post">
 					<?php settings_fields( $this->module->options_group_name ); ?>
-					<?php do_settings_sections( $this->module->options_group_name ); ?>	
+					<?php do_settings_sections( $this->module->options_group_name ); ?>
 					<?php echo '<input id="edit_flow_module_name" name="edit_flow_module_name" type="hidden" value="' . esc_attr( $this->module->name ) . '" />'; ?>
 					<?php submit_button(); ?>
 				</form>
-				<?php else: ?>
-				<?php /** Custom form for adding a new Usergroup **/ ?>
+				<?php else : ?>
+					<?php /** Custom form for adding a new Usergroup **/ ?>
 					<form class="add:the-list:" action="<?php echo esc_url( $this->get_link() ); ?>" method="post" id="addusergroup" name="addusergroup">
 					<div class="form-field form-required">
 						<label for="name"><?php _e( 'Name', 'edit-flow' ); ?></label>
-						<input type="text" aria-required="true" id="name" name="name" maxlength="40" value="<?php if ( !empty( $_POST['name'] ) ) echo esc_attr( $_POST['name'] ); ?>" />
+						<input type="text" aria-required="true" id="name" name="name" maxlength="40" value="<?php /* phpcs:ignore Generic.ControlStructures.InlineControlStructure.NotAllowed */ if ( ! empty( $_POST['name'] ) ) echo esc_attr( $_POST['name'] ); ?>"
```

Comment:
```suggestion
						<input type="text" aria-required="true" id="name" name="name" maxlength="40" value="<?php echo esc_attr( isset( $_POST['name'] ) && ! empty( $_POST['name'] ) ? $_POST['name'] : '' );
```

Reply:
Same suggestion to ternary the two other `phpcs:ignore`s below.

Reply:
Done